### PR TITLE
Couple of small fixes

### PR DESCRIPTION
**Pull request by mrocklin**
1.  Avoid long blockdims repr
2.  Add Array.nbytes property

### CODE REVIEW

File: dask/array/core.py [hunk 1]
```diff
@@ -596,6 +597,10 @@ def _get_block(self, *args):
     def ndim(self):
         return len(self.shape)
 
+    @property
+    def nbytes(self):
```
Comment (cowlicks) [inline dask/array/core.py]: Could you add a docstring?

File: dask/array/core.py [hunk 2]
```diff
@@ -596,6 +597,10 @@ def _get_block(self, *args):
...
+        return np.prod(self.shape) * self.dtype.itemsize
```
Comment (shoyer) [inline dask/array/core.py]: This would be a good idea to add `size` as a property, too

Comment (mrocklin) [inline dask/array/core.py]: Fixed

File: dask/array/core.py (hunk 1 above)
Comment (mrocklin) [inline dask/array/core.py]: Fixed

File: dask/array/core.py [hunk 3]
```diff
@@ -596,6 +597,16 @@ def _get_block(self, *args):
...
         return len(self.shape)
 
+    @property
+    def size(self):
+        """ Number of bytes in array """
+        return np.prod(self.shape)
+
+    @property
+    def nbytes(self):
+        """ Number of elements in array """
```
Comment (shoyer) [inline dask/array/core.py]: You mixed up the docstrings for size and nbytes

Comment (mrocklin) [inline dask/array/core.py]: Fixed

**Comment by mrocklin** [comment]
Merging soon if no comment

**Comment by cowlicks** [comment]
LGTM

**Pull request merged**